tsdb/head_test.go

```diff
@@ -5420,22 +5252,21 @@ func TestCuttingNewHeadChunks(t *testing.T) {
 				numSamples int
 				numBytes   int
 			}{
-				{30, 696},
-				{30, 700},
-				{30, 708},
-				{30, 693},
+				{40, 896},
+				{40, 899},
+				{40, 896},
+				{30, 690},
 				{30, 691},
-				{30, 692},
-				{30, 695},
 				{30, 694},
+				{30, 693},
```

Comment:
Looks like these changes in the encoded chunks were triggered by fixing how the `observationCount` is calculated when generating those synthetic histograms:
https://github.com/prometheus/prometheus/pull/12977/commits/2acb51ecda54747315c16560a6d39f1b697b4ad5#diff-7c1dfea5c568897587621510c04d322f908a8ed40751e592d63d2f8422b6a78dL4961-R4969